tools/remote_build/linux_kokoro.bazelrc

```diff
@@ -12,22 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# bazelrc file for running gRPC tests on Kokoro using Foundry
+# bazelrc file for running gRPC tests on Kokoro using RBE (on Linux)
+# This file has extra settings for tests that run on CI (kokoro).
 
-import %workspace%/tools/remote_build/rbe_common.bazelrc
-
-build --remote_cache=grpcs://remotebuildexecution.googleapis.com
-build --remote_executor=grpcs://remotebuildexecution.googleapis.com
-
-build --auth_enabled=true
-
-build --bes_backend=grpcs://buildeventservice.googleapis.com
-build --bes_timeout=600s
-build --project_id=grpc-testing
+import %workspace%/tools/remote_build/linux.bazelrc
 
 # required by kokoro for some reason
+# TODO(jtattermusch): is this still actually required?
 build --test_env=USER=anon
 
 build --jobs=200
-build --test_output=errors
-build --keep_going=true
```

Comment:
`keep_going` is removed?

Reply:
Yes, I think it's not useful.  We're already running "bazel test" and by default all the tests are run (and bazel won't stop at first failed test), so `--keep_going` isn't very useful since that's already implied by needing to run all the tests (minus maybe some cornercases, but I don't think we care about them).